### PR TITLE
Fix for Issue #271

### PR DESCRIPTION
**Pull request by BenjaminHolland**
Small change, but may help if we're still wanting to get performance testing back up.

### CODE REVIEW

**Comment by BenjaminHolland** [comment]
Also starting small seems like a good idea. I'm still getting used to the whole "Legit Project I Don't Control" workflow.

**Comment by seanlip** [comment]
Thanks! Please remove console.logs. You can see a diff of what's changing by clicking on the "Files Changed" tab in GitHub: https://github.com/google/tie/pull/272/files -- it's a good idea to make sure that's correct and exactly what you intend, before getting a review.

(Also, passing to @eyurko for review, once you've got this PR finalized.)

**Comment by BenjaminHolland** [comment]
Whoops! Forgot I'd done that.

File: client/services/code_preprocessors/PythonCodePreprocessorService.js [hunk 1]
```diff
@@ -131,9 +134,11 @@ tie.factory('PythonCodePreprocessorService', [
             break;
           }
           if (lastFunctionNameLocation !== functionNameStart) {
+            
             codeToAdd = functionPrefix;
+            var functionAdded=codeToAdd+code.slice(lastFunctionNameLocation,functionName.length);
```
Comment (seanlip) [inline client/services/code_preprocessors/PythonCodePreprocessorService.js]: Is this used?

**Comment by seanlip** [comment]
Hi @BenjaminHolland, thanks! As a general rule, could you also try to make sure the diff is as minimal as possible? In other words, the stuff in the "Files changed" tab should contain a few changes from time.time to time.clock, and nothing else.

**Comment by BenjaminHolland** [comment]
I'll check that in the future. A lot of times Im doing commits in quick
breaks during other projects, so I don't always check everything before the
commit as closely as i should.

File: client/services/code_preprocessors/PythonCodePreprocessorService.js (hunk 1 above)
Comment (eyurko) [inline client/services/code_preprocessors/PythonCodePreprocessorService.js]: If it is, please add spaces around the + and after the ,

File: client/services/code_preprocessors/PythonCodePreprocessorService.js [hunk 2]
```diff
@@ -274,12 +281,13 @@ tie.factory('PythonCodePreprocessorService', [
...
           outputFunctionName, correctnessTests, buggyOutputTests,
           performanceTests) {
+            
         // Transform the student code (without changing the number of lines) to
         // put it within a class.
         var transformedStudentCode = this._transformCodeToInstanceMethods(
           codeSubmission.getRawCode(), CLASS_NAME_STUDENT_CODE);
         codeSubmission.replace(transformedStudentCode);
-
+
```
Comment (eyurko) [inline client/services/code_preprocessors/PythonCodePreprocessorService.js]: trailing whitespace

**Comment by BenjaminHolland** [comment]
Regarding the referenced waiting pulls, would you prefer each one be done individually or have them merged into a single one?

**Comment by seanlip** [comment]
Ah -- please see comment in https://github.com/google/tie/issues/277#issuecomment-297873487. Each should be done individually (especially if they're for unrelated things) so that they can be reviewed separately, but you can have multiple branches open at once.

**Pull request merged**